data/bunrodea/bunrodea missions.txt

```diff
@@ -205,15 +205,15 @@ mission "First Contact: Bunrodea (Hostile)"
+			`	The Queen motions to one of the guards and says something to them that the creature does not translate. She then speaks to you, and the creature translates: "Since you have apologized, you will be allowed to live. My guards have been ordered to leave you be, and you may come and go on a small number of our worlds. But this is only so that you do not cause further trouble, and to allow you to refuel if you are passing through my systems. Your presence will only be tolerated, not welcomed. Now please leave my court."`
 				goto end
 			
 			label threatened
-			`	The Queen motions to one of the guards and says something to it that the creature does not translate. She then speaks to you, and the creature translates: "Since I have shown you mercy, you will be allowed to live. My guards have been ordered to leave you be, and you may come and go on a small number of our worlds. But this is only so that you do not cause further trouble, and to allow you to refuel if you are passing through my systems. Your presence will only be tolerated, not welcomed. Now please leave my court."`
+			`	The Queen motions to one of the guards and says something to them that the creature does not translate. She then speaks to you, and the creature translates: "Since I have shown you mercy, you will be allowed to live. My guards have been ordered to leave you be, and you may come and go on a small number of our worlds. But this is only so that you do not cause further trouble, and to allow you to refuel if you are passing through my systems. Your presence will only be tolerated, not welcomed. Now please leave my court."`
 				goto end
 			
 			label slighted
-			`	The Queen motions to one of the guards and says something to it that the creature does not translate. She then speaks to you, and the creature translates: "Since it is my guards that have shown you disrespect, you will be allowed to live. My guards have been ordered to leave you be, and you may come and go on a small number of our worlds. But this is only so that you do not cause further trouble, and to allow you to refuel if you are passing through my systems. Your presence will only be tolerated, not welcomed. Now please leave my court."`
+			`	The Queen motions to one of the guards and says something to them that the creature does not translate. She then speaks to you, and the creature translates: "Since it is my guards that have shown you disrespect, you will be allowed to live. My guards have been ordered to leave you be, and you may come and go on a small number of our worlds. But this is only so that you do not cause further trouble, and to allow you to refuel if you are passing through my systems. Your presence will only be tolerated, not welcomed. Now please leave my court."`
```

Comment:
Technically the guards could justifiably be "it" - we have established that these are insectoids with a queen, I don't think know if the guards actually have personhood.

Reply:
I'd prefer `they`, but let's ask @Amazinite.

Reply:
I think humans would implicitly assume all the individual creatures are bestowed with personhood- it's a cultural difference you can confront later.

Reply:
I personally would not assume that the unspeaking bug warriors - who only "speak" robotically with cybernetic implants in a specific entry point to the mission, of what appears to be an insectoid caste-based alien race with a Queen - were in possession of personhood.